reconcile/sentry_config.py

```diff
@@ -0,0 +1,356 @@
+import logging
+import reconcile.queries as queries
+import utils.gql as gql
+import utils.secret_reader as secret_reader
+
+from reconcile.github_users import init_github
+from utils.config import get_config
+from utils.sentry_client import SentryClient
+
+
+SENTRY_PROJECTS_QUERY = """
+{
+  apps: apps_v1 {
+    sentryProjects {
+      team {
+        name
+        instance {
+          consoleUrl
+        }
+      }
+      projects {
+        name
+        description
+        email_prefix
+        platform
+        sensitive_fields
+        safe_fields
+        auto_resolve_age
+        allowed_domains
+      }
+    }
+  }
+}
+"""
+
+SENTRY_USERS_QUERY = """
+{
+  roles: roles_v1 {
+    name
+    users {
+      name
+      github_username
+    }
+    bots {
+      name
+      github_username
+    }
+    sentry_teams {
+      name
+      instance {
+        consoleUrl
+        automationToken {
+          path
+          field
+          format
+          version
+        }
+      }
+    }
+  }
+}
+"""
+
+SENTRY_TEAMS_QUERY = """
+{
+  teams: sentry_teams_v1 {
+    name
+    instance {
+      consoleUrl
+      automationToken {
+        path
+        field
+        format
+        version
+      }
+    }
+  }
+}
+"""
+
+SENTRY_INSTANCES_QUERY = """
+{
+  instances: sentry_instances_v1 {
+    consoleUrl
+    automationToken {
+      path
+      field
+      format
+      version
+    }
+    adminUser {
+      path
+      field
+      format
+      version
+    }
+  }
+}
+"""
+
+class SentryState:
+  def __init__(self):
```

Comment:
Indentation has to be a multiple of 4.

Reply:
There are many others flake8 errors...

Reply:
Done

Reply:
Not a fan of this requirement though. :)

Reply:
That's PEP8: https://www.python.org/dev/peps/pep-0008/#indentation
We don't have to follow, but we would need a reason for that... and to change it everywhere else :)